README.md

```diff
@@ -77,10 +77,12 @@ For details on the `json` and `ldap` backends see the [documentation](https://ow
         -p 127.0.0.1:389:389 \
         -p 636:636 -d osixia/openldap
     ```
+1.  start a Redis server e.g. with docker:
+    `docker run -e REDIS_DATABASES=1 -p 6379:6379 -d webhippie/redis:latest`
 
-2.  clone oC10 code: `git clone https://github.com/owncloud/core.git`
+1.  clone oC10 code: `git clone https://github.com/owncloud/core.git`
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[ordered-list-marker-value] Marker should be `3`, was `1`](https://app.codacy.com/gh/owncloud/ocis-reva/pullRequest?prid=4997850)